File: LinkedList/src/main/java/org/mainProgramme/Queue/Stack.java
package org.mainProgramme.Queue;

public class Stack<T> {
    Deque<T> store = new Deque<>();

    // Push item to Top / Bottom
    public void Push(T item) { store.EnqueueHead(item);}

    // Pop Item out (Pop top)
    public T Pop() { return store.DequeueHead(); }

    // Peek Top Item
    public T Peek() { return store.peekHead(); }

}

File: Graph/Stack.py
class Stack:
    def __init__(self):
        self.stack = []
    def push(self, node):
        self.stack.append(node)
    def pop(self):
        top = self.stack[len(self.stack) - 1]
        del self.stack[len(self.stack) - 1]
        return top
    def isEmpty(self) -> bool:
        return len(self.stack) < 1
    def peek(self):
        return self.stack[len(self.stack) - 1]
